pkg/chartutil/requirements.go

```diff
@@ -106,3 +116,144 @@ func LoadRequirementsLock(c *chart.Chart) (*RequirementsLock, error) {
 	r := &RequirementsLock{}
 	return r, yaml.Unmarshal(data, r)
 }
+
+// ProcessRequirementsConditions disables charts based on condition path value in values
+func ProcessRequirementsConditions(reqs *Requirements, cvals Values) {
+	var cond string
+	var conds []string
+
+	if reqs != nil && len(reqs.Dependencies) > 0 {
+		for _, r := range reqs.Dependencies {
+			var hasTrue, hasFalse bool
+			cond = string(r.Condition)
+			// check for list
+			if len(cond) > 0 {
+				if strings.Contains(cond, ",") {
+					conds = strings.Split(strings.TrimSpace(cond), ",")
+				} else {
+					conds = []string{strings.TrimSpace(cond)}
+				}
+				for _, c := range conds {
+					if len(c) > 0 {
+						// retrieve value
+						vv, err := cvals.PathValue(c)
+						if err == nil {
+							if vv.(bool) {
+								hasTrue = true
+							}
+							if !vv.(bool) {
+								hasFalse = true
+							}
+						} else {
+							if _, ok := err.(ErrNoValue); !ok {
+								// this is a real error
+								log.Printf("Warning: PathValue returned error %v", err)
+							}
+						}
+						if vv != nil {
+							// got first value, break loop
+							break
+						}
+					}
+				}
+				if !hasTrue && hasFalse {
+					r.Enabled = false
+				} else {
+					if hasTrue {
+						r.Enabled = true
+					}
+				}
+			}
+
+		}
+	}
+}
+
+// ProcessRequirementsTags disables charts based on tags in values
+func ProcessRequirementsTags(reqs *Requirements, cvals Values) {
+	vt, err := cvals.Table("tags")
+	if err != nil {
+		return
+
+	}
+	if reqs != nil && len(reqs.Dependencies) > 0 {
+		for _, r := range reqs.Dependencies {
+			if len(r.Tags) > 0 {
+				tags := r.Tags
+
+				var hasTrue, hasFalse bool
+				for _, k := range tags {
+					if b, ok := vt[k]; ok {
+						if b.(bool) {
+							hasTrue = true
+						}
+						if !b.(bool) {
```

Comment:
do we really need to evaluate all the statements if at least one has false? I assume , will act as "and" operator, so we can do "for i in conditions; if !condition: return false; endif; return true"